Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/display/FhirDisplayController.java

```diff
@@ -41,6 +41,7 @@
 public class FhirDisplayController implements DisplayController {
 
     private static final Logger LOG = LoggerFactory.getLogger(FhirDisplayController.class);
+    private static final WebServiceProxyHelper HELPER = new WebServiceProxyHelper();
```

Comment:
name convention: use proxyHelper

Reply:
agree.  HELPER is too generic of a variable name.